packages/core/src/common/keybinding.ts

```diff
@@ -132,50 +144,83 @@ export class KeybindingRegistry {
                 return;
             }
         }
-        const { keyCode, commandId } = binding;
```

Comment:
why did you remove this?

Reply:
I just thought that the characters saved by using local variables weren't really worth using the indirection.  But I can add it back if you prefer.

Reply:
No, it's ok. But it is a matter of taste (both ways ok) so I would not have touched it.